typings/index.d.ts

```diff
@@ -3301,8 +3300,12 @@ declare module 'discord.js' {
 
   type MessageAdditions = MessageEmbed | MessageAttachment | (MessageEmbed | MessageAttachment)[];
 
+  type MessageActionRowComponent = MessageButton;
+
+  type MessageActionRowComponentOptions = MessageButtonOptions;
+
   interface MessageActionRowOptions extends BaseMessageComponentOptions {
-    components?: MessageComponentResolvable[];
```

Comment:
Why is this type defined after its first usage?

Reply:
Order doesn't matter.

Reply:
No I meant like, wouldn't it be better regarding how readable the code is?

Reply:
They're defined in alphabetical order, which makes them very easy to find. Any decent editor will also let you jump to a type's definition.

Reply:
They're defined in alphabetical order, which makes them very easy to find. Any decent editor will also let you jump to a type's definition.